spec/api-session-spec.js

```diff
@@ -587,7 +587,7 @@ describe('session module', () => {
       server.listen(0, '127.0.0.1', () => {
         const config = { pacScript: `http://127.0.0.1:${server.address().port}` }
         customSession.setProxy(config, () => {
-          customSession.resolveProxy('http://localhost', (proxy) => {
+          customSession.resolveProxy('https://google.com', (proxy) => {
```

Comment:
⚠️ our tests shouldn't rely on external network connections

Reply:
This doesn't hit the network, only verifies the result from proxy resolver. Also `localhost` cannot be used with PAC tests because of https://chromium-review.googlesource.com/c/chromium/src/+/1324880